packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/TabItem/tabItem.js

```diff
@@ -0,0 +1,3 @@
+const TabItem = ({ children }) => children;
```

Comment:
This _is redundant_ but following our content type structure in Page Builder this helps provide clarity around how the data is structured. Also in a future extensibility model this could be overriding / extended to add additional functionality if needed.